packages/appstore/src/constants/currency-to-icon-mapper.ts

```diff
@@ -0,0 +1,54 @@
+export const currency_to_icon_mapper: Record<string, Record<'light' | 'dark', string>> = {
```

Comment:
refer this https://github.com/binary-com/deriv-app/pull/10003/files#r1318827695

Reply:
@heorhi-deriv I'm not sure about currency_to_icon_mapper and getWalletCurrencyIcon to be only in appstore package. I mean we use the same in useWalletsList and useWalletAccountsList hooks (inside of them), so maybe would be better to create these functions in @deriv/utils, because we have access there from appstore and from hooks packages. Just a question :) You can ignore it